eldritch/events.py

```diff
@@ -1641,6 +1642,12 @@ def resolve(self, state):
       self.closed = True
       state.gates.append(state.places[self.location_name].gate)  # TODO: take a gate trophy
       state.places[self.location_name].gate = None
+      closed_until = state.places[self.location_name].closed_until or -1
+      if closed_until > state.turn_number:
+        state.event_stack.append(
+            CloseLocation(self.location_name, closed_until - state.turn_number - 1)
+        )
+        return False
```

Comment:
Not keeping track of whether you've re-closed the location is slightly dangerous - if anyone were to add more code underneath the `if closed_until > state.turn_number` block, that code wouldn't be executed in the case where the location was re-closed. I would keep a boolean at the class level.

Reply:
To be clear, you're saying don't use a @property for CityPlace.closed, and instead just have it be a boolean like in my original implementation?